app/resources/locale/en/en.po

```diff
@@ -796,129 +793,127 @@ msgstr "New"
 msgid "New Color:"
-msgstr ""
+msgstr "New Color:"
 
 #: app/views/design.html:80
-#, fuzzy
-msgid "New Name:"
-msgstr "Name"
+msgid "New Name"
+msgstr "New Name"
+
+#: app/views/design.html:71
+msgid "Next"
+msgstr "Next"
 
-#: app/scripts/controllers/menu.js:1436
+#: app/scripts/controllers/menu.js:1453
 msgid "No collections stored"
 msgstr "No collections stored"
 
-#: app/scripts/app.js:58 app/scripts/controllers/menu.js:585
-#: app/scripts/services/project.js:146
+#: app/scripts/app.js:57 app/scripts/controllers/menu.js:601
+#: app/scripts/services/project.js:157
 msgid "OK"
 msgstr "OK"
 
 #: app/scripts/services/forms.js:427
-#, fuzzy
 msgid "Olive Drab"
-msgstr "OliveDrab"
+msgstr "Olive Drab"
 
-#: app/views/design.html:208
+#: app/views/design.html:206
 msgid "OliveDrab"
 msgstr "OliveDrab"
 
 #: app/views/menu.html:100
 msgid "Open"
 msgstr "Open"
 
-#: app/scripts/services/utils.js:697
+#: app/scripts/services/utils.js:692
 msgid "Open SVG"
 msgstr "Open SVG"
 
 #: app/scripts/services/forms.js:403
-#, fuzzy
 msgid "Orange Red"
-msgstr "OrangeRed"
+msgstr "Orange Red"
 
-#: app/views/design.html:144
+#: app/views/design.html:143
 msgid "OrangeRed"
 msgstr "OrangeRed"
 
-#: app/scripts/services/project.js:544
+#: app/scripts/services/project.js:555
 msgid "Original file {{file}} does not exist"
 msgstr "Original file {{file}} does not exist"
 
-#: app/views/design.html:288 app/views/menu.html:764
+#: app/views/design.html:284 app/views/menu.html:782
 msgid "Output"
 msgstr "Output"
 
-#: app/views/design.html:300 app/views/menu.html:773
+#: app/views/design.html:294 app/views/menu.html:791
 msgid "Output label"
 msgstr "Output label"
 
 #: app/scripts/services/forms.js:1350
-#, fuzzy
 msgid "Output labels"
-msgstr "Output label"
+msgstr "Output labels"
 
 #: app/scripts/services/forms.js:1109
-#, fuzzy
 msgid "Output port name"
-msgstr "Output label"
+msgstr "Output port name"
 
 #: app/scripts/services/forms.js:1572
-#, fuzzy
 msgid "Output ports"
-msgstr "Output"
+msgstr "Output ports"
 
-#: app/views/design.html:306 app/views/menu.html:778
+#: app/views/design.html:299 app/views/menu.html:796
 msgid "Paired labels"
-msgstr ""
+msgstr "Paired labels"
 
 #: app/views/menu.html:268
 msgid "Paste"
 msgstr "Paste"
 
-#: app/scripts/controllers/menu.js:731 app/scripts/controllers/menu.js:788
-#: app/scripts/controllers/menu.js:853 app/scripts/controllers/menu.js:917
+#: app/scripts/controllers/menu.js:751 app/scripts/controllers/menu.js:808
+#: app/scripts/controllers/menu.js:873 app/scripts/controllers/menu.js:937
 msgid "Path {{path}} does not exist"
 msgstr "Path {{path}} does not exist"
 
-#: app/scripts/services/utils.js:1142
+#: app/scripts/services/utils.js:1137
 msgid "Please run: {{cmd}}"
 msgstr "Please run: {{cmd}}"
 
-#: app/views/menu.html:334
+#: app/views/menu.html:352
 msgid "Preferences"
 msgstr "Preferences"
 
-#: app/views/menu.html:422
+#: app/views/design.html:68
+msgid "Prev."
+msgstr "Prev."
```

Comment:
Maybe we should go with 'Prev' without the dot? In the actual text

Reply:
It's an abbreviation... 🤷🏽‍♂️